specification/netapp/resource-manager/Microsoft.NetApp/preview/2017-08-15/netapp.json

```diff
@@ -1130,6 +1157,49 @@
           "type": "string",
           "readOnly": true,
           "description": "Azure lifecycle management"
+        },
+        "activeDirectories": {
+          "description": "Active Directories",
+          "type": "array",
+          "$ref": "#/definitions/activeDirectories"
+        }
+      }
+    },
+    "activeDirectories": {
+      "description": "Active Directories",
+      "type": "object",
+      "properties": {
+        "activeDirectoryId": {
+          "type": "string",
+          "description": "Id of the Active Directory"
+        },
+        "username": {
+          "type": "string",
+          "description": "Username of Active Directory domain administrator"
+        },
+        "password": {
```

Comment:
Could you elaborate on this parameter? Is this secure?

Reply:
It's credentials for the AD admin at the lower level. The RP only passes this through to the lower level. It is transmitted via https to the RP. But yes quite correct, the security is in question and there is an internal ticket to address this for the next release (pre-GA). Of course this swagger only reflects this implementation.